biz.aQute.launchpad/src/aQute/launchpad/package-info.java

```diff
@@ -1,2 +1,2 @@
-@org.osgi.annotation.versioning.Version("1.2.0")
+@org.osgi.annotation.versioning.Version("1.2.1")
```

Comment:
why a micro change? this is api change.

Reply:
This is required because the string constant changed value.